schemas/pom.xml

```diff
@@ -58,10 +58,42 @@
             <artifactId>java-hamcrest</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>5.2.0</version>
+            <scope>test</scope>
+        </dependency><!-- https://mvnrepository.com/artifact/com.shazam/shazamcrest -->
```

Comment:
Is this comment necessary/on the right line?

Reply:
Didn't these get removed the other day?

Reply:
yes they got removed on 7th august, but if they are not there, then the tests do not run.

Reply:
Comment removed